jvm-packages/xgboost4j-example/src/main/java/ml/dmlc/xgboost4j/java/example/EarlyStopping.java

```diff
@@ -29,9 +29,9 @@
 public class EarlyStopping {
   public static void main(String[] args) throws IOException, XGBoostError {
     DataLoader.CSRSparseData trainCSR =
-        DataLoader.loadSVMFile("../../demo/data/agaricus.txt.train?format=libsvm");
+        DataLoader.loadSVMFile("../../demo/data/agaricus.txt.train");
     DataLoader.CSRSparseData testCSR =
-        DataLoader.loadSVMFile("../../demo/data/agaricus.txt.test?format=libsvm");
+        DataLoader.loadSVMFile("../../demo/data/agaricus.txt.test");
```

Comment:
so, the format has been set to libsvm by default in libxgboost?

Reply:
`DataLoader` is a utility class used only in the example. Users still have to specify `?format=libsvm` when constructing a DMatrix.